### PR TITLE
docs: add manual uninstall instructions

### PR DESCRIPTION
**Pull request by jmg-duarte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,5 +48,11 @@ Aliases for those who use [git-extras](https://github.com/tj/git-extras) (will b
 * ```git rf "commit message here"``` -> ```git commit -m 'refactor: commit message here'```
 * ```git ch "commit message here"``` -> ```git commit -m 'chore: commit message here'```
 
+## (Manual) Uninstall
```
Comment (fteem) [inline README.md:51]:
```suggestion
## Uninstallation
```

File: README.md [hunk 2]
```diff
@@ -48,5 +48,11 @@ Aliases for those who use [git-extras](https://github.com/tj/git-extras) (will b
...
+
+You can manually uninstall `git-semantic-commits` by:
+* Going to `~/.gitconfig` and removing the added aliases.
```
Comment (fteem) [inline README.md:54]:
```suggestion
* Removing the added aliases from `~/.gitconfig`.
```

File: README.md [hunk 3]
```diff
@@ -48,5 +48,11 @@ Aliases for those who use [git-extras](https://github.com/tj/git-extras) (will b
...
+* Going to `.basrc` and removing the line added to the `$PATH` variable.
```
Comment (fteem) [inline README.md:55]:
```suggestion
* Removing the line added to the `$PATH` variable from `~/.bashrc`.
```

**Review by fteem — CHANGES_REQUESTED** [review]
Thanks for adding the improvements. 🙏 I left a couple of suggestions for a better sentence structure / phrasing.

**Comment by jmg-duarte** [comment]
I've applied your suggestions!

**Comment by fteem** [comment]
Thank you for your contribution! 🙏

**Pull request merged**